lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -116,7 +116,7 @@ def initialize(workspace, ns, klass, instance, object_name = nil)
 
       Benchmark.realtime_block(:inherit_time) do
         # Who do we inherit from
-        @inherits = @aec.inherits.blank? ? BASE_OBJECT : @aec.inherits
+        @inherits = BASE_OBJECT
```

Comment:
@Fryguy 
We are only inheriting from the BASE_OBJECT, we are not inheriting from user defined classes, which has been empty in newer databases, the inherits value was only defined in some of the legacy XML databases from old customers.